CODIGO/Declares.bas

```diff
@@ -717,10 +721,32 @@ Global OtroInventario(1 To MAX_INVENTORY_SLOTS) As Inventory
 
 Public UserHechizos(1 To MAXHECHI) As Integer
 
+Public Type PjCuenta
+    nombre      As String
+    Head        As Integer
+    Body        As Integer
+    shield      As Byte
```

Comment:
The variables should start with uppercase to maintain a normalization around the code?

Reply:
If we do so, we should change the name in other places. Because VB will replace them